parsec/core/sync_monitor.py

```diff
@@ -231,7 +231,7 @@ async def tick(self) -> float:
                 del self._local_changes[entry_id]
                 try:
                     await self._sync(entry_id)
-                except FSBackendOfflineError as exc:
+                except (FSBackendOfflineError, BackendConnectionError) as exc:
```

Comment:
BackendConnectionError != BackendNotAvailable
for instance we could have a `BackendConnectionRefused` because our device has been revoked (or because our organization has expired)

So I don't think we should do this conversion, especially given `BackendNotAvailable` is a subclass of `BackendConnectionError`, hence calling code expecting  `BackendNotAvailable` should also expects `BackendConnectionError` (but, of course, does have to expect the Spanish inquisition :trollface: )